lasagne/layers/recurrent.py

```diff
@@ -8,10 +8,12 @@
 The following recurrent layers are implemented:
 
 .. autosummary::
+    :nosignatures:
```

Comment:
It renders the function table without signaures. You can also see updates.py in lasagne. 
Sphinx truncates the signatues so they are useless anyway.